mesheryctl/pkg/utils/helpers.go

```diff
@@ -257,7 +263,30 @@ func PreReqCheck(subcommand string, focusedContext string) error {
 			}
 		}
 	} else if currCtx.Platform == "kubernetes" {
+		if err != nil {
+			return errors.Wrap(err, "failed to create new client")
+		}
+
+		var kubeconfig string
+		home := homedir.HomeDir()
+		if home != "" {
+			kubeconfig = filepath.Join(home, ".kube", "config")
+		}
 
+		config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
+		if err != nil {
+			return err
+		}
+
+		clientset, err := kubernetes.NewForConfig(config)
+		if err != nil {
+			return err
+		}
+
+		_, err = clientset.CoreV1().Pods("").List(context.TODO(), metav1.ListOptions{})
+		if err != nil {
+			return errors.New("kubernetes is not running yet")
+		}
```

Comment:
A log and a return error would be good which is consistent with the current implementation for the docker platform...